npm compat (#431)
denoland/manual: node/how_to_with_npm/mysql.md

# How to use MySQL with Deno

MySQL is the most popular database in the
[2022 Stack Overflow Developer Survey](https://survey.stackoverflow.co/2022/#most-popular-technologies-database)
and counts Facebook, Twitter, YouTube, and Netflix among its users.

You can manipulate and query a MySQL database with Deno using the `mysql2` node

> package and the `npm:` modifier. This allows us to use its Promise wrapper and

Reviewer: ```suggestion
package and importing via `npm:mysql2`. This allows us to use its Promise wrapper and
```

I wouldn't really consider it a modifier.

take advantage of top-level await.

```tsx

> import mysql from "npm:mysql2/promise";

Reviewer: ```suggestion
import mysql from "npm:mysql2@2.3/promise";
```
